spot-client/src/index.js

```diff
@@ -40,23 +40,32 @@ import { ExternalApiSubscriber } from 'spot-remote/external-api';
 import App from './app';
 
 const queryParams = new URLSearchParams(window.location.search);
+const startParams = {};
+
+for (const [ key, value ] of queryParams.entries()) {
+    startParams[key] = value;
+}
 
 const store = createStore(
     ReducerRegistry.combineReducers(reducers),
-    {
+    defaultsDeep({}, getPersistedState(), {
         config: {
             ...setDefaultValues(window.JitsiMeetSpotConfig)
         },
+        setup: {
+            startParams
+        },
         spotTv: {
 
             // Will get overridden on Spot-Remote by Spot-TV updates.
             electron: isElectron()
-        },
-        ...getPersistedState()
-    },
+        }
+    }),
     MiddlewareRegistry.applyMiddleware(thunk)
 );
 
+window.spotStore = store;
```

Comment:
there's already a global for the store--window.spot.

Reply:
that was left there accidentally, removed it